src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1260,6 +1260,24 @@
   <data name="DoNotUseStackallocInLoopsMessage" xml:space="preserve">
     <value>Potential stack overflow. Move the stackalloc out of the loop.</value>
   </data>
+  <data name="PreferStreamReadAsyncMemoryOverloadsTitle" xml:space="preserve">
+    <value>Prefer Memory&lt;Byte&gt; overload for ReadAsync.</value>
```

Comment:
If we are planning to add VB support, then wouldn't use of `Memory<Byte>` in the user facing resource strings be too much C# specific? Shouldn't the VB one have `Memory(Of(Byte))`? If am fine with the current approach, if you have approval from members of your team. If not, it should be easy to use language language specific resource strings + diagnostic descriptors in the analyzer - both the diagnostic descriptors can have identical fields, except the resource strings.

Reply:
You should be able to just make this a placeholder value and do `parameterType.ToDisplayString(SymbolDisplayFormat.CSharpErrorMessageFormat)`, which (despite its name) seems to be "C# format for C# and VB format for VB".

Reply:
That is a valid approach for the diagnostic message, but title and description need to be provided without format specifiers to the diagnostic descriptor ctor.

Reply:
@mavasani, I would like to address this once I submit a PR to address Visual Basic. The Analyzer class is currently topped with the CSharp attribute only.

Reply:
Addressed. The message is now language agnostic.